tests/api-mongo.test.js

```diff
@@ -412,3 +416,26 @@ describe('MongoDB Words', function () {
     });
   });
 });
+
+describe('API Requests For Home Directory "/"', () => {
+  describe('/undefinedRoute', () => {
+    it('should return response status of 404', (done) => {
+      const route = '/undefinedRoute';
+      getApiUrlRoute(route).end((_, res) => {
```

Comment:
Could you move `.end()` to its own line? Other tests do this, and it's great to make sure that they all stay consistent.